### ISSUE TITLE
Background not appear appear when it save

### ISSUE THREAD

**Issue by kentangtelo** [JideGuru/flutter_drawing_board#13]
When i test the project, try add shape or line in backgroud. After i saved it and check the images, the shape or line is appear, but the background or image that added not appear. I want to fixed that but the backgroundImage not null when added images, make me really confuse

https://user-images.githubusercontent.com/72773420/219107601-64e14815-15f8-4518-86e6-67686e190c07.mp4

**Comment by JideGuru** [comment]
Ohh wow. It seems we didn't take note of the export when we added the image import feature.
I'll look into it and try to get it fixed.

**Comment by kentangtelo** [comment]
The demo link in Readme, the export is goes well.

**Comment by JideGuru** [comment]
Yes, you're right. It seems the issue only exists in debug mode. Can you tell me the Flutter version you're using?
